api/src/main/java/org/pmiops/workbench/api/RuntimeController.java

```diff
@@ -409,6 +435,21 @@ private Runtime getOverrideFromListRuntimes(String googleProject) {
     return ResponseEntity.ok(new RuntimeLocalizeResponse().runtimeLocalDirectory(targetDir));
   }
 
+  private void enforceComputeSecuritySuspension(DbUser user) throws FailedPreconditionException {
+    Optional<Instant> suspendedUntil =
+        Optional.ofNullable(user.getComputeSecuritySuspendedUntil()).map(Timestamp::toInstant);
+    if (suspendedUntil.isPresent() && clock.instant().isBefore(suspendedUntil.get())) {
+      throw new FailedPreconditionException(
+          new ErrorResponse()
+              .errorCode(ErrorCode.COMPUTE_SECURITY_SUSPENDED)
+              .message("user is suspended from compute for security reasons")
+              .parameters(
+                  new SecuritySuspendedErrorParameters()
+                      // Instant.toString() yields ISO-8601
```

Comment:
optional nit: 
Do you want ISO-8601? And how UI will utilize this? 
I see two other options: 
1: Like AAR, and AccessModules, return `suspendUntilEpochMillis`, [UI knows how to parse this from number](https://github.com/all-of-us/workbench/blob/fd80c2c4ab567a58b64335d0774dfe53084a0069/ui/src/app/utils/index.tsx#L303)
2: Format it in API, `DateTimeFormatter.ISO_INSTANT.format(suspendedUntil.get())`, but I am not sure if  which zone is good for all users. 
It can also be:
`suspendedUntil.get().atZone(ZoneOffset.UTC).format(DateTimeFormatter.ISO_INSTANT);`

Reply:
Thanks for explaination! SGTM